monkey/monkey_island/cc/resources/agent_configuration_schema.py

```diff
@@ -0,0 +1,20 @@
+from http import HTTPStatus
+
+from monkey_island.cc.resources.AbstractResource import AbstractResource
+from monkey_island.cc.resources.request_authentication import jwt_required
+from monkey_island.cc.services import ConfigSchemaService
+
+
+class AgentConfigurationSchema(AbstractResource):
+    urls = ["/api/agent-configuration-schema"]
+
+    def __init__(self, config_schema_service: ConfigSchemaService):
+        self._config_schema_service = config_schema_service
+
+    @jwt_required
+    def get(self):
+        try:
+            schema = self._config_schema_service.get_schema()
+            return schema, HTTPStatus.OK
+        except RuntimeError as err:
+            return {"error": str(err)}, HTTPStatus.INTERNAL_SERVER_ERROR
```

Comment:
Hmm, why do we only catch RuntimError? How is it different from not handling the error at all?

Reply:
That's the only thing that's raised from `ConfigSchemaService.get_schema()` right now. It'll be changed based on whatever that function finally raises.